geomesa-nifi-processors/src/main/scala/org/geomesa/nifi/geo/AbstractGeoIngestProcessor.scala

```diff
@@ -212,82 +211,82 @@ abstract class AbstractGeoIngestProcessor(dataStoreProperties: Seq[PropertyDescr
       if (value == null) { Seq.empty } else {
         Seq(p.getName -> value)
       }
-    }.toMap
-    val props = configured ++ static // note: static props take precedence
-    logger.trace(s"DataStore properties: $props")
+    }
+    val props = configured.toMap ++ static // note: static props take precedence
```

Comment:
This is just a style change, right?  (e.g. having the .toMap on a line rather than dangling around?)

Reply:
yeah